worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -67,6 +67,14 @@ default CuboidRegion getBoundingBox() {
      */
     Vector3 getCenter();
 
+    /**
+     * Divides the region into N sections and returns a Set with all unique the points in between those sections.
```

Comment:
this description is extremely misleading - it divides the *diagonal of the bounding box* and returns a *list* of points (which might only be unique in fractional parts).